src/test/resources/features/EditWidget.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Edit widget
+
+  Background: User Navigates to the Dashboards Page
+    Given User can navigate to the Dashboards Page
+    And User have chosen a Dashboard
+
+  Rule: Edit widget name
+    Background:
+      Given User is on a Dashboard Page
+
+    Scenario Outline: User edits widget to set new valid widget name
+      Given Widget is visible
+        | type | LAUNCH_STAT            |
+        | name | LAUNCH STATISTICS AREA |
+      When User presses Edit Widget button
+      And enters new name "<widgetName>"
+      Then widget name is changed to new name
+      And can be changed to default name
+        | name | LAUNCH STATISTICS AREA |
```

Comment:
Why do you use data table? You have only one variable. Try this - can be changed to default name "LAUNCH STATISTICS AREA"